libs/core/src/lib/grid-list/components/grid-list/grid-list.component.html

```diff
@@ -9,7 +9,7 @@
     [class.fd-grid-list--mode-single-select-right]="selectionMode === 'singleSelectRight'"
     [class.fd-grid-list--mode-multi-select]="selectionMode === 'multiSelect'"
 >
-    <div class="fd-container" role="listbox">
+    <div class="fd-container" role="listbox" [attr.aria-label]="ariaLabel">
```

Comment:
I'm not sure the container should have the listbox role. This role is applied in a div that holds elements with role option.In your case the container has also the title bar. 
The element with this role (listbox) needs `aria-activedescendant` and/or `aria-multiselectable` and the element with role option that is selected should have aria-selected.
Please check here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role
Note: I am not an a11y expert so don't take my word for 100% true. That's just what I understood from reading the docs. The people who logged the issue should be able to explain in details what you are supposed to do.

Reply:
I agree. However, I didn't do related changes are they're in the scope of another issue (please refer https://github.com/SAP/fundamental-ngx/pull/5643#discussion_r648608796)